.github/workflows/Tests.yml

```
@@ -856,14 +858,28 @@
       VCS_COMMIT_ID: ${{ needs.check_mats.outputs.build_sha }}
       CODECLIMATE_REPO_TOKEN: ${{ secrets.CODECLIMATE_REPO_TOKEN }}
       CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+    outputs:
+      integration_outcome: ${{ steps.integration_outcome.outcome }}
+      integration_test_outcome: ${{ steps.integration_tests.integration_test_outcome }}
```

actionlint (1.7.7)

property "integration_test_outcome" is not defined in object type {conclusion: string; outcome: string; outputs: {string => string}}

(expression)